packages/react-components/src/components/SendBox.tsx

```diff
@@ -183,6 +183,12 @@ export interface SendBoxProps {
    * @beta
    */
   onCancelFileUpload?: (fileId: string) => void;
+  /* @conditional-compile-remove(file-sharing) */
+  /**
+   * Optional arialabel strings for file upload cards
+   * @beta
+   */
+  fileUploadStrings?: _FileUploadCardsStrings;
```

Comment:
`strings` prop already exists on SendBox.
Let's utilize it for introducing more strings.

Reply:
done